src/Digdir.Domain.Dialogporten.Application/Features/V1/ResourceRegistry/Commands/Synchronize/SynchronizeResourceRegistryCommand.cs

```diff
@@ -0,0 +1,142 @@
+using Digdir.Domain.Dialogporten.Application.Common.ReturnTypes;
+using Digdir.Domain.Dialogporten.Application.Externals;
+using Digdir.Domain.Dialogporten.Domain.SubjectResources;
+using MediatR;
+using Medo;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using OneOf;
+using OneOf.Types;
+
+namespace Digdir.Domain.Dialogporten.Application.Features.V1.ResourceRegistry.Commands.Synchronize;
+
+public class SynchronizeResourceRegistryCommand : IRequest<SynchronizeResourceRegistryResult>
+{
+    public DateTimeOffset? Since { get; set; }
+}
+
+[GenerateOneOf]
+public partial class SynchronizeResourceRegistryResult : OneOfBase<Success, ValidationError>;
+
+internal sealed class SynchronizeResourceRegistryCommandHandler : IRequestHandler<SynchronizeResourceRegistryCommand, SynchronizeResourceRegistryResult>
+{
+    private readonly IDialogDbContext _db;
+    private readonly IUnitOfWork _unitOfWork;
+    private readonly IResourceRegistry _resourceRegistry;
+    private readonly ILogger<SynchronizeResourceRegistryCommandHandler> _logger;
+
+    public SynchronizeResourceRegistryCommandHandler(
+        IDialogDbContext db,
+        IUnitOfWork unitOfWork,
+        IResourceRegistry resourceRegistry,
+        ILogger<SynchronizeResourceRegistryCommandHandler> logger)
+    {
+        _db = db ?? throw new ArgumentNullException(nameof(db));
+        _unitOfWork = unitOfWork ?? throw new ArgumentNullException(nameof(unitOfWork));
+        _resourceRegistry = resourceRegistry ?? throw new ArgumentNullException(nameof(resourceRegistry));
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+    }
+    public async Task<SynchronizeResourceRegistryResult> Handle(SynchronizeResourceRegistryCommand request, CancellationToken cancellationToken)
+    {
+        // 1. Get the last updated timestamp from parameter, or the database, or use a default
+        var lastUpdated = request.Since
+            ?? (await _db.SubjectResourceLastUpdates
+                .OrderByDescending(x => x.LastUpdate)
+                .FirstOrDefaultAsync(cancellationToken))
+                ?.LastUpdate
+            ?? DateTimeOffset.MinValue;
+
+        // Allow for a few seconds of drift between server clocks
+        var newLastUpdated = DateTimeOffset.Now.Subtract(TimeSpan.FromSeconds(3));
```

Comment:
En lite annoyance med det API-et som ble implementert er at en oppgit since-verdi vil inkludere alle med dét timestampet, så en vil ende opp med å få det samme changesettet gang på gang. En workaround er kanskje å legge på ett nanosekund, siden to distinkte changesets i praksis alltid vil legge lengre mellom hverandre i tid.

Men hovedsaklig tenker jeg det er en fordel å ha SubjectResourceLastUpdates siden det gir muligheten til å enkelt gjøre en full resynk (eller devis siden et gitt tidspunkt).